### PR TITLE
using live replay binlogs by default.

### PR DESCRIPTION
**Pull request by mialinx**
until timestamp may be used as deadline for live replay

### CODE REVIEW

File: internal/databases/mysql/mysql.go [hunk 1]
```diff
@@ -193,7 +194,7 @@ outer:
 				break outer
 			}
 		}
-		if len(logsToFetch) == 0 || !liveReplay {
+		if len(logsToFetch) == 0 || now.After(endTS) {
```
Comment (ostinru) [inline internal/databases/mysql/mysql.go:197]: There are 2 condition to exit outer loop:
1) no new files = `len(logsToFetch) == 0`
OR
2) binlog timestamp is after endTS

Second check is on line 193, so I believe that we can remove this `now.After(endTS)` check

Comment (mialinx) [reply internal/databases/mysql/mysql.go]: fixed

**Comment by ostinru** [comment]
👍

**Review by ostinru — APPROVED** [review]
👍

**Pull request merged**